go.mod

```diff
@@ -170,6 +170,7 @@ require (
 
 require (
 	github.com/akrylysov/pogreb v0.10.1
+	github.com/cometbft/cometbft v0.0.0-00010101000000-000000000000
```

Comment:
Is there a reason for this version? It seems quite old: https://pkg.go.dev/github.com/cometbft/cometbft?tab=versions